supabase/functions/_shared/Redis.ts

```
@@ -39,6 +41,29 @@ export class Redis {
+        // Then check the underlying Upstash client
+        const clientAsAny = target.client as unknown as Record<string, unknown>;
+        const clientMethod = clientAsAny[String(prop)];
+        
+        if (typeof clientMethod === "function") {
+          // Bind the method to the client so 'this' works correctly
+          return clientMethod.bind(target.client);
+        }
+        
+        return undefined;
+      }
+    });
```

Biome (2.1.2)

[error] The constructor should not return a value.

The constructor is here:

Returning a value from a constructor may confuse users of the class.

(lint/correctness/noConstructorReturn)